.pre-commit-config.yaml

```diff
@@ -29,7 +29,7 @@ repos:
     -   id: isort
         language_version: python3
         exclude: ^(build/*)|(docs/*)|(examples/*)
-        args: [-w120, -m3, --tc, --project=gpytorch]
+        args: [-w120, -m3, --tc, --project=gpytorch, --profile=black]
```

Comment:
BTW. You will want this regardless. Sometimes black & isort can get in an infinite loop fight where they cancel each other's changes. This flag tells isort to use black conventions.

Reply:
An alternative to this would be to use µfmt: https://ufmt.omnilib.dev/en/stable/ - we've had very good experiences using this. Ofc this is orthogonal to the changes here.